app/config/routing.yml

```diff
@@ -1,31 +1,39 @@
 wallabag_annotation:
   type : rest
   resource: "@WallabagAnnotationBundle/Resources/config/routing_annotations.yml"
+  prefix: "%base_path%/"
 
 wallabag_import:
     resource: "@WallabagImportBundle/Controller/"
     type: annotation
-    prefix: /import
+    prefix: "%base_path%/import"
 
 wallabag_api:
     resource: "@WallabagApiBundle/Resources/config/routing.yml"
-    prefix: /
+    prefix: "%base_path%/"
 
 app:
     resource: "@WallabagCoreBundle/Controller/"
     type: annotation
+    prefix: "%base_path%/"
 
 doc-api:
     resource: "@NelmioApiDocBundle/Resources/config/routing.yml"
-    prefix: /api/doc
+    prefix: "%base_path%/api/doc"
 
 rest :
     type : rest
     resource : "routing_rest.yml"
-    prefix : /api
+    prefix : "%base_path%/api"
+
+homepage_main:
+    path: "%base_path%/"
+    defaults:
+        _controller: WallabagCoreBundle:Entry:showUnread
+        page: 1
```

Comment:
I understand the problem but do we have to handle homepage url with trailing slash?